src/main/java/net/minecraftforge/common/crafting/ConditionalAdvancement.java

```diff
@@ -36,16 +36,14 @@ public static Builder builder()
     @Nullable
     public static JsonObject processConditional(JsonObject json, ICondition.IContext context) {
         JsonArray entries = GsonHelper.getAsJsonArray(json, "advancements", null);
-        if (entries == null)
-        {
-            return CraftingHelper.processConditions(json, "conditions", context) ? json : null;
-        }
+        if (entries == null) return json;
```

Comment:
What does this change?

Reply:
The root conditions would have already been processed by `SimpleJsonResourceReloadListener`, checking them here as well would mean that they'd be checked twice.

Reply:
Are there any cases where a modder may be calling this method manually? Given it is public and static